python/lsst/daf/persistence/posixStorage.py

```diff
@@ -601,6 +606,22 @@ def search(root, path, searchParents=False):
             else:
                 return None
 
+    @staticmethod
+    def storageExists(uri):
```

Comment:
This method is added but nothing in this commit seems to call it. Is it a specialization of a base implementation?